deeplake/api/dataset.py

```diff
@@ -811,7 +812,7 @@ def rename(
         """Renames dataset at ``old_path`` to ``new_path``.
 
         Examples:
-
+            >>> deeplake.rename("hub://username/image_ds", "new_ds")
             >>> deeplake.rename("hub://username/image_ds", "hub://username/new_ds")
```

Comment:
@nvoxland To me this doesn't make sense unless you do a copy. Changing an org means moving the storage, at least in come cases. Therefore, how do you rename the entire path?

I feel like the rename methods should only change the name hub://org/name

Reply:
In that example, the new name is the dataset within the org. 

So `deeplake.rename("hub://username/image_ds", "new_ds")` is a shorter/safer version  of `deeplake.rename("hub://username/image_ds", "hub://username/new_ds")` 

I could take that example (and the little code added to support it) out. But it seemed like a "rename" operation should just be giving the new name in general, rather than a whole path which may be something much too different (like having a different org_id or something)

Whatever you like

Reply:
@nvoxland What happens if the org is changed:

deeplake.rename("hub://org1/image_ds", "hub://org_new/new_ds")

Reply:
It throws a value exception